spec/routing/documents_routing_spec.rb

```diff
@@ -33,31 +33,31 @@
 
 describe DocumentsController do
   describe "routing" do
-    it { get('/projects/567/documents').should route_to(:controller => 'documents',
+    it { expect(get('/projects/567/documents')).to route_to(:controller => 'documents',
```

Comment:
Line is too long. [87/80]<br>Avoid using {...} for multi-line blocks.<br>Use the new Ruby 1.9 hash syntax.<br>Block body expression is on the same line as the block start.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.